internships/gsoc2022.md

```diff
@@ -1,80 +1,115 @@
+
+During the interview:
+- we will ask you to give a short (5 to 10 minutes; to be finalised) presentation on a piece of Python code that you wrote
+- we will ask about your motivation to join sktime, your previous experience, your suitability for the specified project, and you general technical background in data science and Python.
+
+After the interview, we will rank the candidates and send a list of our preferred candidates to GSoC, who will make the final decisions and allocations.
+We will inform you of *our* perspective on the interviews (see below for possible outcomes) at most one week after the final candidate is interviewed, and GSoC will inform you of the final decision by May 20.
+
+When we inform you of the outcome of the interview, there are three possibilities:
+* Conditional fast-track acceptance. This means we put you as one of the top three ranking candidates, in our list sent to GSoC. This means highly likely - but not guaranteed - acceptance to the sktime GSoC program.
+* Conditional acceptance. This means you are in our list sent to GSoC.  This means likely - but not guaranteed - acceptance to the sktime GSoC program.
+* A rejection. This means you are not in our list sent to GSoC for 2022.
+  - We understand this will be disheartening for you, but this is only a rejection for GSoC, which is particularly demanding of participants. We will always welcome anybody interested in joining sktime outside of GSoC.
+
+## What we expect from you
+GSoC is a marathon, not a sprint, and we expect good performance over the whole project.
+
+Our expectations for GSoC participants before GSoC starts:
+ * You are interested in time series, machine learning, statistics, API design and software architecture.
+ * You like coding in Python.
+ * You are familiar with the basic data science ecosystem in Python, including numpy, pandas and scikit-learn.
+ * You enjoy working with a vibrant team of experienced ML scientists and software engineers.
+ * You are enthusiastic about open-source.
+
+Our expectations for GSoC participants during GSoC:
+* You follow our [Code of Conduct](https://www.sktime.org/en/stable/get_involved/code_of_conduct.html).
+* You work full time on your GSoC project.
+* You maintain daily contact with your mentor(s).
+* You engage with the sktime community and other GSoC participants.
+* You peer-review a fellow student's work in the middle and at the end of GSoC.
+* You write weekly blog posts and a final summary post at the end of the project.
+* You consider becoming a *long-term* developer and contributor, stepping up to become a leader in open source toolbox development.
+
+## What you can expect from us
+The expectations on you are high, but you can expect just as much from us. You should expect:
+* We follow the [Code of Conduct](https://www.sktime.org/en/stable/get_involved/code_of_conduct.html).
+* 1-1 mentoring with an experienced contributor.
```

Comment:
perhaps add, weekly individual mentoring meeting, max two working day latency in reaction from core devs, daily standups with the community

Reply:
done